### PR TITLE
ref(update): remove depricated ensureRepo func

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

**Comment by mboersma** [comment]
There is still an [`ensureRepo`](https://github.com/deis/helm/blob/master/config/config.go#L172) func in config/config.go that's nearly identical and is being called. Should we take the test from update_test.go and add it to config_test.go instead of deleting it?

**Comment by michelleN** [comment]
@mboersma yeah that's a good call. I'll refactor.

File: config/config_test.go [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+func createTmpHome() string {
+	tmpHome, _ := ioutil.TempDir("", "helm_home")
+	defer os.Remove(tmpHome)
+	return tmpHome
+}
+
+func TestEnsureRepo(t *testing.T) {
+	tmpHome := createTmpHome()
+
+	repo := "https://github.com/deis/charts"
```
Comment (bacongobbler) [inline config/config_test.go]: now we're located at helm/charts :)

Comment (michelleN) [inline config/config_test.go]: alrighty! thanks.

**Comment by michelleN** [comment]
rebased and ready :)

**Pull request merged**